contents/components/radio-buttons.html

```diff
@@ -139,15 +139,30 @@ <h3>Selected / Unselected</h3>
   <div class="cf nr2 nl2 mb3">
     <div class="fl-ns w-50-ns ph2">
       <img src="../images/components/radio-buttons/radiobutton-states.svg" alt="Example of a selected and an unselected radio button." id="showImage" style="height:202px;">
-      <iframe class="db w-100" height="202.5" src="https://designakt.github.io/PDS/components/radio-buttons/behavior.html" allowfullscreen="allowfullscreen" frameborder="0" id="showExample" style="display: none; margin-bottom: 15px;"></iframe>
-      <small class="db grey-50 mb3 mb0-ns">
-         <a href="#" onclick="document.getElementById('showImage').style.display='block';
-                              document.getElementById('showExample').style.display='none';
-                              return false;">Image</a> /
-         <a href="#" onclick="document.getElementById('showImage').style.display='none';
-                              document.getElementById('showExample').style.display='block';
-                              return false;">Interactive example</a> (Currently only renders correctly in Firefox.)
-      </small>
+      <div class='interactive'>
```

Comment:
Here it should probably be `<div class="interactive">` instead of `<div class='interactive'>`, so double quotes instead of single quotes